.github/workflows/protobuf.yaml

```diff
@@ -23,12 +23,12 @@ env:
 
 jobs:
   compatibility:
-    runs-on: ubuntu-latest
+    runs-on: [ubuntu-22.04-github-hosted-16core]
```

Comment:
Just to clarify: Is this change somehow related to decomposing the `schema` crate, or is it a general change that would be performed anyway?